src/challenges/challenge_status.cpp

```diff
@@ -78,14 +86,28 @@ void ChallengeStatus::setSolved(RaceManager::Difficulty d)
     {
         m_state[curr] = CH_SOLVED;
     }
-}
+} // setSolved
+
+// ------------------------------------------------------------------------
+bool ChallengeStatus::isUnlockList()
+{
+    return m_data->isUnlockList();
+} // isUnlockList
+
+// ------------------------------------------------------------------------
+bool ChallengeStatus::isGrandPrix()
+{
+    return m_data->isGrandPrix();
```

Comment:
single-line methods can be specific in the header file to help the compiler with inlining